src/main/scala/cognite/spark/v1/SdkV1Relation.scala

```diff
@@ -88,6 +90,17 @@ abstract class SdkV1Relation[A <: Product, I](config: RelationConfig, shortName:
       val rowOfAllFields = toRow(item)
       Row.fromSeq(indicesOfRequiredFields.map(idx => rowOfAllFields.get(idx)))
     }
+
+  // scalastyle:off no.whitespace.after.left.bracket
+  def updateByExternalIdHandler[
```

Comment:
`updateByExternalId` is better, that's what it does and the name of the method it's using. I think it's clearer. "handler" is usually used for callbacks, and this isn't that.